### DISCUSSION TITLE
VirusTotal reports spyware/malicious from multiple engines

### DISCUSSION THREAD

**Discussion by sitte** [rustdesk/rustdesk#2418] [General]
New RustDesk 1.2.0 has many detection of malware/malicious (Avast, AVG, Kaspersky ...) :
https://www.virustotal.com/gui/file/99e00fb57ca0c81442c27053a519cd2bb6314191fdec08a724e6cec09f574899
Is it safe?

**Comment by trekfan1** [comment]
i have already reported this, and the thread is closed saying "the file is not signed yet", i m not found the 3d, probably deleted.....
